.docker/scripts/init-ext.sh

```diff
@@ -4,11 +4,33 @@
 #
 set -e
 
+install_requirements () {
+    PROJECT_DIR="$1"
+    for filename in requirements-$PYTHON_VERSION.txt requirements.txt pip-requirements.txt; do
+        if [ -f "$PROJECT_DIR/$filename" ]; then
+            pip install -r "$PROJECT_DIR/$filename"
+            return 0
+        fi
+    done
+}
+
+install_dev_requirements () {
+    PROJECT_DIR="$1"
+    for filename in dev-requirements-$PYTHON_VERSION.txt requirements-dev-$PYTHON_VERSION.txt requirements-dev.txt dev-requirements.txt; do
```

Comment:
should we look at doing them all in one line so it can work itself out? (i.e. build a list of existing files and their order?

Reply:
Not really. We don't want to install versioned and unversioned requirement files at the same time, because they're usually incompatible. Eg CKAN has `requirements.txt` and `requirements-py2.txt`, and the unversioned one is meant for Python 3+, so you don't want to install both.